### DISCUSSION TITLE
User input from gradle task?

### DISCUSSION THREAD

**Discussion by Mr-EmPee** [microsoft/vscode-gradle#1414] [Q&A]
Currently when running a task with the gradle plugin I can't write anything inside the terminal of the running task, is there an option to fix this?

**Comment by jdneo** [comment]
That terminal is a pseudo terminal. If you want to run tasks with arguments, you can right-click a task and select `Run Task with Args`

![image](https://github.com/microsoft/vscode-gradle/assets/6193897/6b071ff3-74d6-4fa5-87fd-d95421883d37)

**Reply by Mr-EmPee** [reply]
I've a task that requires user input from the terminal it doesn't accept args

**Reply by jdneo** [reply]
Filed an issue here: https://github.com/microsoft/vscode-gradle/issues/1416
